README.md

```diff
@@ -58,5 +41,118 @@ You can also track upcoming changes using the [awaiting-deployment](https://gith
+latest 2 versions of an OS.
+
+## Image Releases
+*How to best follow along with changes*
+
+1. Find the latest releases for this repository [here.](https://github.com/actions/virtual-environments/releases)
+2. Subscribe to the releases coming out of this repository, instructions [here.](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository)
+3. Upcoming changes: A pre-release is created when deployment is started. As soon as deployment is finished, it is converted to a release. If you have subscribed to releases, you will get notified of pre-releases as well.
+   - You can also track upcoming changes using the [awaiting-deployment](https://github.com/actions/virtual-environments/labels/awaiting-deployment) label.
+4. For high impact changes, we will post these in advance to the GitHub Changelog on our [blog](https://github.blog/changelog/) and on [twitter](https://twitter.com/GHchangelog).
+   - Ex: breaking changes, GA or deprecation of images
+5. You can also view notable upcoming changes by viewing the [Announcement](https://github.com/actions/virtual-environments/labels/Announcement) label.
+
+*Cadence*
+
+- We typically deploy weekly updates to the software on the runner images.
+
+## Software and Image Support
+
+### Support Policy
+- Tools and versions will typically be removed 6 months after they are deprecated or have reached end-of-life
+- We support(at maximum) 2 GA images and 1 beta image at a time. We begin the deprecation process of the oldest image label once the newest OS image label has been released to GA.
+- The images generally contain the latest versions of packages installed except for Ubuntu LTS where we rely on the Canonical-provided repositories mostly.
+- Popular tools can have several versions installed side-by-side with the following strategy:
+
+| Tool name | Installation strategy |
+|-----------|-----------------------|
+| Docker images | not more than 3 latest LTS OS\tool versions. New images or new versions of current images are added using the standard tool request process |
+| Java      | all LTS versions |
+| Node.js   | 3 latest LTS versions |
+| Go        | 3 latest minor versions |
+| Python <br/> Ruby | 5 most popular `major.minor` versions |
+| PyPy      | 3 most popular `major.minor` versions |
+| .NET Core | 2 latest LTS versions and 1 latest version. For each feature version only latest patch is installed |
+| GCC <br/> GNU Fortran <br/> Clang <br/> GNU C++ | 3 latest major versions |
+| Android NDK | 1 latest non-LTS, 2 latest LTS versions |
+| Xcode     | - all OS compatible versions side-by-side <br/> - for beta, GM versions - latest beta only <br/> - old patch versions are deprecated in 3 months |
+
+### Image Deprecation Policy

```

Comment:
Even with this content, I would recommend adding the definition to the image definition section.

Reply:
i'm curious what you think the value of that would be? Originally considered creating a "deprecated" definition but ultimately @thejoebourneidentity decided against it since it felt unnecessary.